SPECS/squashfs-tools/squashfs-tools.spec

```diff
@@ -17,6 +17,8 @@ BuildRequires:  xz-devel
 BuildRequires:  zlib-devel
 BuildRequires:  zstd-devel
 
+%global major_ver 4.6
```

Comment:
can you script this, so it does not need to be manually changed for the next upgrade?